apps/web/src/services/checkout-service.ts

```diff
@@ -141,6 +141,16 @@ export class CheckoutService implements CheckoutAPI {
       .json<AdminPaymentList>()
   }
 
+  async getPaymentsByBankAccountId(
+    bankAccountId: string
+  ): Promise<AdminPaymentList> {
+    return await this.http
+      .get(
+        `${urls.api.v1.admin.getPaymentsForBankAccount}?bankAccountId=${bankAccountId}`
+      )
+      .json<AdminPaymentList>()
+  }
+
```

Comment:
Should this be in the admin service? I was conflicted where to place these functions. The admin middleware is used for these endpoints, so maybe it'd be worth keeping in admin.

Reply:
I would think so, yes.